backend/src/common/types/email.type.ts

```diff
@@ -0,0 +1,3 @@
+type Email = { text: string } | ({ html: string } & { subject?: string });
```

Comment:
What is the purpose of this `({ html: string } & { subject?: string })` construction with `&`? Maybe it can be simplified like this `{ html: string; subject?: string }`?

Reply:
oh, here is a little mistake, it should looks like this:  type Email = ({ text: string } | { html: string }) & { subject?: string };

Reply:
oh, then this makes sense now

Reply:
fixed